content/guide/drupal/index.md

```diff
@@ -42,7 +42,7 @@ You can install LoginRadius  Drupal Plugin [manually](#install-manually) or via
 
 ### Install via Web Interface
 
-1. Download the <a href="https://github.com/LoginRadius/drupal-identity-module/blob/master/drupal-8/package/customer_identity_and_access_management.tar" target="_blank">Drupal v8.x</a> or <a href="https://github.com/LoginRadius/drupal-identity-module/blob/master/drupal-9/package/customer_identity_and_access_management.tar" target="_blank">Drupal v9.x</a> Customer Identity module.
+1. Download the <a href="(https://github.com/LoginRadius/drupal-identity-module/blob/master/drupal-8/package/customer_identity_and_access_management.tar" target="_blank">Drupal v8.x</a> or <a href="https://github.com/LoginRadius/drupal-identity-module/blob/master/drupal-9/package/customer_identity_and_access_management.tar" target="_blank">Drupal v9.x</a> Customer Identity module.
```

Comment:
@saintmalik 
Thank you for your contribution.
You have made the change at the wrong place, it is working fine.

You can fix the broken link under the "**Install Manually**" heading (just need to remove the  "(" **start bracket** )

Reply:
Also, raise the PR for dev branch not for main

Reply:
okay thanks, will do that

Reply:
@saintmalik please let me know if you are fixing this.

Reply:
i will be fixing it, on it